src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,13 +1,21 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private int bladeLength;
+    private int ripperLength;
+    private String bulldozerType;
```

Comment:
It would be better to use just `type`:
```suggestion
    private String type;
```